ext/node/ops/crypto/digest.rs

```diff
@@ -99,6 +99,19 @@ impl Hash {
       Sha512(context) => context.finalize(),
     }
   }
+
+  pub fn get_hashes() -> Vec<&'static str> {
+    vec![
+      "md4",
+      "md5",
+      "ripemd160",
+      "sha1",
+      "sha224",
+      "sha256",
+      "sha384",
+      "sha512",
+    ]
+  }
```

Comment:
Is there a particular reason to move this code to Rust? I think it would be fine to keep this function in JavaScript, since it's a list that's static. Seems like an unnecessary overhead to cross JS/Rust boundary to get it

Reply:
Good point! The reason I chose to keep it in the Rust part is the closeness to the source of all crypto algorithms. Since they are defined and implemented in the `digest.rs` file, people wouldn't forget to update the list when adding/removing new ones. Does this makes sense?

Reply:
Sure, I guess we can do that